packages/bpk-component-input/src/BpkInput-test.js

```diff
@@ -65,6 +65,48 @@ describe('BpkInput', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('should render correctly with "clearable" attribute', () => {
+    // Will report a proptypes error because 'onClear' and 'clearButtonLabel'
```

Comment:
Having a comment for the reason of having the jest spy it's brilliant, should a rule!